views/travelData.js

```diff
@@ -0,0 +1,67 @@
+const axios = require("axios");
+const querystring = require("querystring");
+const moment = require("moment");
+const API_URL = "https://intermodal.router.hereapi.com/v8/routes";
+
+function getTravelData(req, res) {
+  let queryStr = req.params.query;
+  let queryObj = querystring.parse(queryStr);
+  let origin = queryObj.origin;
+  let dest = queryObj.dest;
+
+  let p = axios.get(API_URL, {
+    params: {
+      apiKey: process.env.api_key,
+      alternatives: 0,
```

Comment:
Set the upper limit of alternatives to 10. Because in the end we'll be comparing all the possible travel options.